src/Agent.Worker/Release/ReleaseDefinitionToFolderMap.cs

```diff
@@ -0,0 +1,10 @@
+using Newtonsoft.Json;
+
+namespace Microsoft.VisualStudio.Services.Agent.Worker.Release
+{
+    public sealed class ReleaseDefinitionToFolderMap
+    {
+        [JsonProperty("ReleaseDirectory")]
```

Comment:
Does this get serialized as camelCase? I believe all json in VSTS typically follows that pattern.